policy.md

```diff
@@ -1,178 +1,116 @@
 
-Audits based on ETSI CPs: Audits conducted by accredited conformity assessment bodies (CAB) must have their Audit Attestation Letter (AAL) uploaded to the CAB’s website. CAs provide the URL to the AAL on the CAB’s website, and ALV will verify those URLs against a known list of AAL locations.
+#### 5.1.1 ETSI 
 
-* When an ETSI Certificate cannot be issued, the CA must still provide an AAL such that there are no gaps between audit periods for consecutive audits. The CA may post the AAL on their own website or attach the attestation report to a [Bugzilla Bug](https://www.ccadb.org/cas/fields#uploading-documents) and provide that URL. Additionally, the CA needs to provide an explanation about the problems and time frame for resolution of the problems.
+Audits conducted by accredited CAB must have their AAL uploaded to the CAB’s website. CAs provide the URL to the AAL on the CAB’s website, and ALV will verify those URLs against a known list of AAL locations.
 
-WebTrust Audits: Audits conducted by licensed WebTrust practitioners must have a WebTrust Seal. CAs enter the URL to the WebTrust Seal into the CCADB, and upon saving of the record, the CCADB automatically converts the URL to point to the corresponding PDF file via integration with CPA Canada.
+When an ETSI Certificate cannot be issued, the CA must still provide an AAL such that there are no gaps between audit periods for consecutive audits. The CA may post the AAL on their own website or attach the attestation report to a [Bugzilla Bug](https://www.ccadb.org/cas/fields#uploading-documents) and provide that URL. Additionally, the CA must provide an incident report for each audit finding (i.e., non-conformity),  and/or problem resulting in the failure of the ETSI Certificate’s issuance, in a Bugzilla Bug within seven calendar days of the audit attestation letter’s issuance date.
 
-* For qualified WebTrust audits, the CA may post the audit statements on their own website or attach the audit statement to a [Bugzilla Bug](https://www.ccadb.org/cas/fields#uploading-documents) and provide that URL. Additionally, the CA needs to provide an explanation about the findings and time frame for resolution of the findings.
+#### 5.1.2 WebTrust
 
-Format Specifications for SHA-256 Fingerprints:
+Audits conducted by licensed WebTrust practitioners must have a WebTrust Seal. CAs enter the URL to the WebTrust Seal into the CCADB, and upon saving of the record, the CCADB automatically converts the URL to point to the corresponding PDF file via integration with CPA Canada.
+
+For qualified WebTrust audits, the CA may post the audit statements on their own website or attach the audit statement to a Bugzilla Bug and provide that URL. Additionally, the CA must provide an incident report for each audit finding (i.e., qualification) in a Bugzilla Bug within seven days of the audit letter’s issuance date.
```

Comment:
New requirement for CCADB SC discussion: "Additionally, the CA must provide an incident report for each audit finding (i.e., qualification) in a Bugzilla Bug within seven days of the audit letter’s issuance date."